js/utils.js

```diff
@@ -473,14 +473,6 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
           return true;
         }
 
-        if (condition === 'between') {
```

Comment:
This doesn't seem to be relevant to `is one of` and looks like we're removing more code than is relevant to the PR title.

Reply:
@ivandevupp Yes please focus the revert on the PR that it's reverting. If you have another PR that needs to be reverted, create a separate PR.